### PR TITLE
Add config file for development environment

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Previously the development environment and production environment share the same config file path, this cause a lot of confusion even for me. This PR adds a config file path for the development environment (`~/.config/wukong/dev/config.toml`). Running wukong without `prod` features, which only happens during development, will use the development environment config file.

Ticket:

### Added
- [x] add config file for development environment

### Changed
- [x] replace `lazy_static` crate with `once_cell` crate since Rust will include `once_cell` API in the future. https://github.com/clap-rs/clap/issues/3828

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/50#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Pull request merged**
